android/run_parallel_test/parallel.conf.js

```diff
@@ -1,10 +1,10 @@
 exports.config = {
   user: process.env.BROWSERSTACK_USERNAME || 'BROWSERSTACK_USERNAME',
-  key: process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_ACC_KEY',
+  key: process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_USERNAME',
```

Comment:
follow naming conventions from this PR https://github.com/kn-neeraj/python-appium-app-browserstack/blob/python3-readme-changes/android/examples/run-first-test/config.json